compiler/rustc_expand/src/mbe/macro_rules.rs

```diff
@@ -651,6 +660,30 @@ fn check_matcher(sess: &ParseSess, def: &ast::Item, matcher: &[mbe::TokenTree])
     err == sess.span_diagnostic.err_count()
 }
 
+fn has_compile_error_macro(rhs: &mbe::TokenTree) -> bool {
+    use mbe::TokenTree::*;
+    match rhs {
+        Delimited(_sp, d) => {
```

Comment:
```suggestion
        TokenTree::Delimited(_sp, d) => {
```
Nit: could you avoid the glob import above?